src/utils/web.py

```diff
@@ -145,7 +147,12 @@ def getUrlFd(url, headers=None, data=None, timeout=None):
             request.add_data(data)
         httpProxy = force(proxy)
         if httpProxy:
-            request.set_proxy(httpProxy, 'http')
+            proxyHandler = ProxyHandler({
+                'http': httpProxy,
+                'https': httpProxy
+                })
+            proxyOpener = build_opener(proxyHandler)
+            install_opener(proxyOpener)
```

Comment:
That installs the proxy globally, until the next call to `getUrlFd`.
So if the config value is changed, this is not applied to plugins not using `getUrlFd`.

Setting the proxy from the config itself would be better.